### PR TITLE
Add lazyimread recipe

### PR DESCRIPTION
**Pull request by lyehe**
Lazyimread is a Python library designed to streamline the handling of large, multi-dimensional image datasets. It offers a unified interface for importing various image file formats, including TIFF, HDF5, Zarr, image sequences, and video files. Lazyimread simplifies the process of working with diverse image data sources by eliminating the need for format-specific boilerplate code.

The library ensures efficient memory usage and flexible dimension order handling. These features make Lazyimread particularly useful for researchers and developers dealing with large variants of imaging data. Additionally, the package provides asynchronous loading capabilities and GUI utilities, further enhancing its versatility in both scripted and interactive environments.

By adding Lazyimread to conda-forge, we aim to make this powerful tool more accessible to the scientific Python community, facilitating easier installation and integration into existing workflows.

### CODE REVIEW

**Comment by lyehe** [comment]
@conda-forge/help-python

Could you please review this new recipe? Thank you!

File: recipes/lazyimread/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Recipe for lazyimread package
```
Comment (synapticarbors) [inline recipes/lazyimread/meta.yaml:1]: Please remove this comment

**Review by synapticarbors — COMMENTED** [review]
Looks good. Happy to merge after removing the comment

File: recipes/lazyimread/meta.yaml (hunk 1 above)
Comment (lyehe) [reply recipes/lazyimread/meta.yaml]: All set

File: recipes/lazyimread/meta.yaml [hunk 2]
```diff
@@ -1,5 +1,3 @@
-# Recipe for lazyimread package
```
Comment (lyehe) [inline recipes/lazyimread/meta.yaml]: Removed

**Pull request merged**